src/app/(auth)/signup/page.tsx

```diff
@@ -114,30 +127,32 @@ export default function SignUp() {
       />
       <Flex $direction="column" $gap="20px">
         <BigBlueButton type="submit">Sign Up</BigBlueButton>
-        <H4Centered>
+        <PCentered>
           Have an account already?{' '}
-          <LinkColored $color={COLORS.greyDark} href="/login">
+          <LinkColored $color={COLORS.blueMid} href={CONFIG.login}>
             Log in
           </LinkColored>
-        </H4Centered>
+        </PCentered>
       </Flex>
     </SmallCardForm>
   ) : (
     <SmallCardForm onSubmit={handleResendEmail}>
-      <Flex $direction="column" $gap="20px">
-        <H2>An email verification link has been sent.</H2>
-        <H4 $color={COLORS.greyDark}>
+      <Flex $direction="column" $align="center" $gap="20px">
+        <Icon type="email" />
+        <AuthSubHeading>Verification email sent!</AuthSubHeading>
+        <H4Centered $color={COLORS.greyDark}>
           This link will direct you to the next step. If you didn’t receive an
           email, please click Resend Email.
-        </H4>
-        <HorizontalDiv>
-          <BigBlueButton type="submit">
-            <H4 $color="white">Resend Email</H4>
-          </BigBlueButton>
-          {emailSentCount > 1 && (
-            <P $color={COLORS.greyMid}>Email has been resent!</P>
-          )}
-        </HorizontalDiv>
+        </H4Centered>
+        <BigBlueButton type="submit">
+          <H4 $color="white">Resend Email</H4>
+        </BigBlueButton>
+        {emailSentCount > 1 && (
+          <P $color={COLORS.greyDark}>
+            Email has been sent to
+            <span style={{ color: COLORS.blueDark }}>{email}</span>
```

Comment:
<img width="569" alt="Screen Shot 2024-05-18 at 11 50 36 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/64336644/dc7dda84-7289-4734-af21-93fd31e2967d">

Can we add an extra space in before the email?